src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatSeparator = fromFormat[fromFormat.length - 1];
```

Comment:
Use [at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)
```suggestion
  const fromFormatSeparator = fromFormat.at(-1);
```

Reply:
at() doesn't work. It shows me an error: "TypeError: fromFormat.at is not a function"